fahrschein-metrics-micrometer/src/main/java/org/zalando/fahrschein/metrics/micrometer/MicrometerMetricsCollector.java

```diff
@@ -0,0 +1,78 @@
+package org.zalando.fahrschein.metrics.micrometer;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import org.zalando.fahrschein.MetricsCollector;
+import org.zalando.fahrschein.MultiplexingMetricsCollector;
+
+import java.util.Arrays;
+
+public class MicrometerMetricsCollector implements MetricsCollector {
+
+    public static final String DEFAULT_PREFIX = name("fahrschein","listener");
```

Comment:
Is it intended that the prefix is different here?

Reply:
intentional, as the existing micrometer integration into the spring-boot-starter already uses this prefix.